contracts/potlock/src/potlock_admin_interactions.rs

```diff
@@ -0,0 +1,111 @@
+use crate::potlock_storage::{self, PotlockId, ProjectId, Status};
+
+multiversx_sc::imports!();
+multiversx_sc::derive_imports!();
+
+pub type ProjectPercentage = MultiValue2<usize, u64>;
+pub const MAX_PERCENTAGE: u64 = 10_000; // 100%
+
+#[multiversx_sc::module]
+pub trait PotlockAdminInteractions:
+    potlock_storage::PotlockStorage + multiversx_sc_modules::only_admin::OnlyAdminModule
+{
+    #[only_admin]
+    #[endpoint(changeFeeForPots)]
+    fn change_fee_for_pots(&self, token_identifier: TokenIdentifier, fee: BigUint) {
+        require!(
+            token_identifier.is_valid_esdt_identifier(),
+            "Invalid token provided"
+        );
+        require!(
+            token_identifier.is_valid_esdt_identifier() && fee.ge(&BigUint::zero()),
+            "Invalid token identifier or amount is 0"
+        );
+        self.fee_token_identifier().set(&token_identifier);
+        self.fee_amount().set(fee);
+    }
+
+    #[only_admin]
+    #[endpoint(acceptPot)]
+    fn accept_pot(&self, potlock_id: PotlockId) {
+        self.require_potlock_exists(potlock_id);
+        let fee_amount = self.fee_amount().get();
+
+        self.fee_amount_accepted_pots()
+            .update(|amount| *amount += fee_amount);
+        let mut accepted_potlock = self.potlocks().get(potlock_id);
+        accepted_potlock.status = Status::Active;
+        self.potlocks().set(potlock_id, &accepted_potlock);
+    }
+
+    #[only_admin]
+    #[endpoint(removePot)]
+    fn remove_pot(&self, potlock_id: PotlockId) {
+        self.require_potlock_exists(potlock_id);
+
+        let potlock_mapper = self.potlocks();
+        let pot_proposer = potlock_mapper.get(potlock_id).proposer;
+        let fee_pot_payment = EsdtTokenPayment::new(
+            self.fee_token_identifier().get(),
+            0u64,
+            self.fee_amount().get(),
+        );
+
+        self.send()
+            .direct_non_zero_esdt_payment(&pot_proposer, &fee_pot_payment);
+        self.potlocks().clear_entry(potlock_id);
+    }
+
+    #[only_admin]
+    #[endpoint(acceptApplication)]
+    fn accept_application(&self, project_id: ProjectId) {
```

Comment:
Maybe also add another endpoint to reject and remove project applications? Like with the pot?